infracost-usage-example.yml

```diff
@@ -186,6 +186,16 @@ resource_usage:
   aws_fsx_windows_file_system.my_system:
     backup_storage_gb: 10000 # Total storage used for backups in GB.
 
+  aws_kinesis_analytics_application.my_kinesis:
+    kinesis_processing_units: 10 # Number of Kinesis processing units.
+
+  aws_kinesisanalyticsv2_application.my_kinesis:
+    kinesis_processing_units: 10 # Number of Kinesis processing units.
+    durable_application_backup_gb: 100 # Total amount of durable application backup in GB.
```

Comment:
we should add this to `aws_kinesis_analytics_application` as it's the same type of resource, just the older version

Reply:
Done